### ISSUE TITLE
Non-standard Peacock 2 Postprocessor Tab Buttons

### ISSUE THREAD

**Issue by friedmud** [idaholab/moose#9123]
### Description of the enhancement or error report

The "save" buttons on the Postprocessor Tabs in Peacock 2 are non-standard and non-intuitive.  It's not clear whether they are "open" buttons or "save" buttons... and they don't look like anything else on OSX.  They also take up quite a bit of vertical space and at least 3 of them `PY`, `</>` and the last one (table?) do not fit any normal icon or idea that a user will have... they are just confusing.

In addition, you did away with the `Open` and `Save` buttons on the `Input File` tab (which I sorely miss)... but for some reason you have some here.  Consistency is key.  They should either all have `Open` and `Save` (and it should say that on the button or use a standard OSX icon to say that) or none of them should.

My suggestion: bring back the `Open` and `Save` buttons on all tabs.  Locate them in the same places on all tabs so it is consistent.  Clicking `Save` on one of the Postprocessor tabs should allow you to choose the file type (as the extension... like normal on OSX).

The two buttons that are about "Showing" stuff should be moved to the menubar.  These are advanced features and don't need dedicated buttons taking up space.

### Rationale for the enhancement or information for reproducing the error
Better usability

### Identified impact
Better usability

**Comment by friedmud** [comment]
Ok - this is still a serious problem.  PLEASE... can we pretty please (with sugar on top) get some normal buttons for saving from the Exodus View, Postprocessor and VecgtorPostprocessor tabs?

It is SUPER non-intuitive... and the popup descriptions don't always work.

I also still maintain that there's no reason for the "Pythong Script Saving" button on these tabs... put that in a menu under Advanced or something... most people won't be doing that often enough to warrant it taking up screen real estate.

**Comment by permcody** [comment]
I agree, this is more important that getting the OS X menus right. Let's get easier to find save and open buttons in Peacock. Thanks

**Comment by permcody** [comment]
OK - we talked about this some more and I was thinking specifically about the open button on the Exodus Viewer tab despite your clear explanation. I dislike that particular button. I'm more neutral on the extra buttons on the Postprocessor and VectorPostprocessor tabs because I don't believe there's a button standard for exporting.

I believe the most standard thing we could do is have an "Export" item on the "File" menu that allows you to select the format you want. Is that reasonable?

**Comment by friedmud** [comment]
Glad to see some movement on this - can  we definitely get some "open"
buttons on the Postprocess and VPP tabs too?

Derek

**Comment by brianmoose** [comment]
@aeslaughter The `Plugin` class has a `addToMainMenu` method that can be used to create new menu items on the main menu. The Input and Execute tab both use this to create their menus.
Maybe not what you had in mind as a "plugin like method" though.
